moped-editor/src/utils/mapHelpers.js

```diff
@@ -7,6 +7,7 @@ import { get } from "lodash";
 
 export const MAPBOX_TOKEN = process.env.REACT_APP_MAPBOX_TOKEN;
 export const drawnLayerName = "drawnByUser";
+const TRAIL_LINE_TYPE = "Off-Street";
```

Comment:
It shouldn't go without saying, I like the color and it makes it easy to the eye and it makes the app look more Kawaii